### ISSUE TITLE
GFM-flavored markdown (newline)?

### ISSUE THREAD

**Issue by elliot-nelson** [sywac/sywac-website#3]
I noticed in a recent PR that I wrote GitHub-style paragraphs (with newlines at 90 characters), thinking they would be formatted as paragraphs.  However, the newlines are being rendered as line breaks within the paragraph.

Typically this is an option you can turn on or off, so I guess this is a request to either:

(a) Turn on github-style paragraph formatting for the markdown renderer.
(b) Confirm that all paragraphs should have no newlines, in which case I can do a follow-up PR to fix the formatting.

(See http://sywac.io/docs/type-properties.html for example of the stilted paragraph formatting.)

**Comment by nexdrew** [comment]
Hmm, interesting, I didn't even notice it 😅 

As you probably already know, I'm using an ancient version of hexo to generate the sywac.io website.

Before we go too far trying to configure it the way we want it, we should probably upgrade hexo or migrate to another option for static site generation.

Just saying.

**Comment by elliot-nelson** [comment]
Makes sense.  For now I'm just attempting to adjust to the "write every paragraph all on one line" style, and there may be no need to change it.  Definitely not before upgrading hexo.
